src/main/java/frc/commands/SuccOffCommand.java

```diff
@@ -0,0 +1,47 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class succOffCommand extends Command {
+  public succOffCommand() {
+    // Use requires() here to declare subsystem dependencies
```

Comment:
need to require subsystem for this to work, my bad
```suggestion
 requires(Robot.vacuum);
    // Use requires() here to declare subsystem dependencies
```